src/FamilyHubs.ReferralUi.Ui/Extensions/StringExtensions.cs

```diff
@@ -8,6 +8,6 @@ public static string ToSentenceCase(this string input)
             return input;
 
         string sentence = input.ToLower();
-        return sentence[0].ToString().ToUpper() + sentence.Substring(1);
+        return string.Concat(sentence[0].ToString().ToUpper(), sentence.AsSpan(1));
```

Comment:
`Concat` works, but is there a reason why we can't use `$""`?

Reply:
I have altered the code as you have suggested. That is what sonarqube suggested